docs/docs/data-structures/overview.mdx

```diff
@@ -62,7 +61,7 @@ Although the value of a pair can be a complex object, the value of any given pai
 
 ## SharedString
 
-The [SharedString](./string) DDS is used for unstructured text data that can be collaboratively edited. It is optimistic.
+The [SharedString](./string.md) DDS is used for unstructured text data that can be collaboratively edited. It is optimistic.
```

Comment:
Should this be .mdx?

Reply:
Nvm, i see that the string file is a .md. Wondering why its not mdx like the right?